conans/requirements_dev.txt

```diff
@@ -4,3 +4,4 @@ parameterized>=0.6.3
 mock>=1.3.0, <1.4.0
 WebTest>=2.0.18, <2.1.0
 bottle
+docker
```

Comment:
the `docker` pypi package is pure python, but it's worth checking whether all the dependencies are also pure python and to estimate the likelihood of dependency conflicts in the future.

otherwise I wonder if we may want to consider making it an optional dependency?

```pip install conan[docker]```